test.py

```diff
@@ -2,13 +2,25 @@
 """
 Some basic tests to verify that the wrapper is working
 """
+from os import getenv
+
+from dotenv import load_dotenv, find_dotenv
```

Comment:
The test.py file is for unit testing that should pass on any machine without the need for anything not passed in with the package. The way that these tests are written, they would require a valid mailchimp username, secret, and access token to be in the system's environment variables when running the tests, and this is not documented anywhere in this pull request. Please add a note to both README files after the 'Installation' section and the 'Upgrading from v1.x' section indicating what is necessary to provide those 3 data points to successfully complete the tests.